grunt/webpack-dev-server.js

```diff
@@ -5,15 +5,19 @@ let clone = require('clone');
 // clone the webpack config to separate configuration of webpack and dev server
 let webpackConfig = clone(require('./webpack').options);
 
-//port for webpack-dev-server
+// port for development server
 const port = +('GA'.split('').reduce((p, c)=> p + c.charCodeAt(), ''));
 
-// enable live reload without a script tag
-webpackConfig.entry.vendor.unshift(`webpack-dev-server/client?http://localhost:${port}`);
```

Comment:
Is there a reason this was removed? Without it, live-reload won't work. Enabling it elsewhere actually puts the dev server client into the vendor bundle, so I think this is the right place for it. See: https://webpack.github.io/docs/webpack-dev-server.html#inline-mode-with-node-js-api